test/manifests/normal/scaleup.yaml

```diff
@@ -0,0 +1,4 @@
+properties:
+  agentPoolProfiles:
+  - name: compute
+    count: 2
```

Comment:
I see you have one file for the scale up partial request scenario. That means we would need another file for the scale down part in the case of a scale up/down e2e test.

For such a test it seems to me that its simpler to just manipulate the full cluster struct before sending it to the RP. Do you have something different in mind as to how this should operate?

Reply:
> For such a test it seems to me that its simpler to just manipulate the full cluster struct before sending it to the RP. Do you have something different in mind as to how this should operate?

@jim-minter can keep me honest but requests landing on the production RP can be partial hence my latest work on merging (eg. on a scale up/down only the agent pool name and count are needed). We want to simulate that behavior and ensure we support partial requests.

Reply:
@kargakis I will integrate the partial updates techniques in my PR. It works for me. My earlier comment was just stating the fact that it would require a little more work from my side.